src/components/Home/components/Header/index.js

```diff
@@ -0,0 +1,93 @@
+import React from "react";
+import classNames from "classnames";
+import header from './styles.css';
+import styles from '../../../../styles/styles.css';
+import bootstrap from 'bootstrap/dist/css/bootstrap.min.css';
+import FontAwesome from "react-fontawesome";
```

Comment:
Maybe we should follow a specific order of imports? I propose components above styles.